src/app/stream-deck/async-module.ts

```diff
@@ -45,60 +63,22 @@ const installFarmingObserver = _.once(() => {
   );
 });
 
-// collect and send data to the stream deck
-function refreshStreamDeck(): ThunkResult {
-  return async (dispatch, getState) => {
-    const refreshAction = () => {
-      const state = getState();
-      const store = currentStoreSelector(getState());
-      if (!store) {
-        return setTimeout(() => dispatch(refreshStreamDeck()), 1000);
-      }
-      sendToStreamDeck({
-        action: 'state',
-        data: {
-          postmaster: packager.postmaster(store),
-          maxPower: packager.maxPower(store, state),
-          vault: packager.vault(state),
-          metrics: packager.metrics(state),
-          equippedItems: packager.equippedItems(store),
-        },
-      });
-    };
-    clearInterval(refreshInterval);
-    refreshInterval = window.setInterval(refreshAction, 30000);
-    refreshAction();
-  };
-}
+// observe inventory state and refresh the stream deck when it changes
+const installInventoryObserver = _.once(() => {
+  observeStore(
+    (state) => state.inventory,
+    (_old, _new, state) => refreshStreamDeck(state),
+  );
+});
```

Comment:
We're moving away from these store observers to a new mechanism, see https://github.com/DestinyItemManager/DIM/pull/10231